### PR TITLE
Add 'repmgr node check' to upgrade preflight checks and auto-tests

### PR DESCRIPTION
**Pull request by to-bar**

### CODE REVIEW

File: data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml [hunk 1]
```diff
@@ -94,25 +94,41 @@
...
-    {{ pg_old.repmgr.bin_dir[ansible_os_family] }}/repmgr \
-      -f {{ pg_old.repmgr.config_dir[ansible_os_family] }}/repmgr.conf \
-      cluster show --csv
-  changed_when: false
-  delegate_to: "{{ groups.postgresql[0] }}"
-  run_once: true
+  vars:
+    _repmgr_config_file: "{{ pg_old.repmgr.config_dir[ansible_os_family] }}/repmgr.conf"
+  block:
+    - name: Check if PostgreSQL {{ pg_old.pg.version }} cluster is healthy
```
Comment (atsikham) [inline data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml:114]: Do we need to check cluster status on every node?

File: data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml [hunk 2]
```diff
@@ -94,25 +94,41 @@
...
+      # when there are two primary nodes (after failover) rc is non-zero
+      command: >-
+        {{ pg_old.repmgr.bin_dir[ansible_os_family] }}/repmgr -f {{ _repmgr_config_file }}
+        cluster show
```
Comment (atsikham) [inline data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml:118]: Will the otput be used somehow? If not, I would stay with --csv as it reduces it.

**Review by atsikham — CHANGES_REQUESTED** [review]

File: data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml (hunk 1 above)
Comment (to-bar) [reply data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml:114]: It's checked only on one node, see line 119 😉

File: data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml (hunk 2 above)
Comment (to-bar) [reply data/common/ansible/playbooks/roles/postgresql/tasks/preflight/upgrade.yml:118]: The csv output is not human friendly at all. I prefer normal for debugging purposes.

**Review by atsikham — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Comment by seriva** [comment]
Can be merged as discussed with @przemyslavic so we can start work on #2666 today. He will test it later from develop.

**Pull request merged**